src/Microsoft.Health.Fhir.SqlServer/Microsoft.Health.Fhir.SqlServer.csproj

```diff
@@ -25,6 +25,11 @@
       <Namespace>Microsoft.Health.Fhir.SqlServer.Features.Schema.Model</Namespace>
       <Args>$([System.IO.Path]::GetFullPath('$(MSBuildThisFileDirectory)\Features\Schema\Migrations\2.sql'))</Args>
     </Generated>
+    <Generated Include="Features\Schema\Model\VLatest.Generated.cs">
```

Comment:
I can't really think of a scenario where we would need both V2 and VLatest classes. When V3 rolls around, we may have code that that conditionally references V2, in which case we would include codegen for it.

Similarly, I don't think we have any code referencing V1 anymore, so we can drop codegen for it.

Reply:
Right, fixed in 9f72df5. That's good to know for [#AB72041](https://microsofthealth.visualstudio.com/Health/_workitems/edit/72041).